J/JWAS/Compat.toml

```diff
@@ -4,30 +4,37 @@ julia = "0"
 ["0-0.2"]
 DataFrames = "0-0.10"
 
-["0.0.2-0"]
-Distributions = "0-0.18"
-
 ["0.0.2-0.0"]
 julia = "0.3-0"
 
+["0.0.2-0.5.3"]
+Distributions = "0-0.18"
+
 ["0.1"]
 julia = "0.4-0"
 
-["0.1.1-0"]
-ProgressMeter = "0"
-
 ["0.1.1-0.1"]
 Plots = "0"
 
+["0.1.1-0.5.3"]
+ProgressMeter = "0"
+
 ["0.2"]
 julia = "0.5-0"
 
 ["0.3"]
 julia = "0.6-0"
 
-["0.3-0"]
+["0.3-0.5.3"]
 CSV = "0"
 DataFrames = "0"
 
-["0.4-0"]
+["0.4-0.5.3"]
 julia = "0.7-1"
+
+["0.5.4-0"]
+CSV = "0.4.3-0"
+DataFrames = "0.18.1-0"
+Distributions = "0.18-0"
+ProgressMeter = "0.9-0"
+julia = "1.1-1"
```

Comment:
Tests seems to pass on Julia 1.0, don't you want this release available there?

Reply:
I'd like to use 1.1 for this version.

Reply:
Why not both? It passes tests on Julia 1.0.

Reply:
Ok. Do I need to update the commit?

Reply:
Update https://github.com/reworkhow/JWAS.jl/blob/dc8efa9cde657ee2d3db9ce8621b0e81dd41be3c/Project.toml#L22 to `julia = "1"` and then retrigger the bot

Reply:
I've changed it to Julia = "1"